deployment/ccip/changeset/v1_5/e2e_test.go

```diff
@@ -48,8 +48,13 @@ func TestE2ELegacy(t *testing.T) {
 				},
 			},
 		}),
-		testhelpers.WithNumOfChains(3),
-		testhelpers.WithChainIDs([]uint64{chainselectors.GETH_TESTNET.EvmChainID}))
+		// testhelpers.WithNumOfChains(3),
+		testhelpers.WithEVMChainsBySelectors([]uint64{
+			chainselectors.GETH_TESTNET.Selector,
+			chainselectors.TEST_90000001.Selector,
+			chainselectors.TEST_90000002.Selector,
+		}),
+	)
```

Comment:
Chaanged to directly specify the chains instead of using a combination of N chains and direct selectors